rdmo/projects/tests/test_viewset_project_contact.py

```diff
@@ -151,7 +151,8 @@ def test_contact_get_set(db, client):
 
 @pytest.mark.parametrize('username,password', users)
 @pytest.mark.parametrize('project_id', projects)
-def test_contact_post(db, client, username, password, project_id):
+def test_contact_post(db, client,settings, username, password, project_id):
+    assert settings.PROJECT_CONTACT
```

Comment:
Why do we need to check this here?

Reply:
it's easier to know what this test is for like that and what the feature flag does..